src/Symfony/Component/DependencyInjection/Loader/YamlFileLoader.php

```diff
@@ -884,6 +884,13 @@ private function resolveServices($value, string $file, bool $isParameter = false
             if ('abstract' === $value->getTag()) {
                 return new AbstractArgument($value->getValue());
             }
+            if ('php/enum' === $value->getTag()) {
```

Comment:
Why adding 2 ways to add enums in Yaml, one with the parser-level support and one with a custom tag ?